client/my-sites/plan-features-2023-grid/components/features.tsx

```diff
@@ -84,17 +106,7 @@ const PlanFeatures2023GridFeatures: React.FC< {
 							<span className={ spanClasses } key={ key }>
 								<span className={ itemTitleClasses }>
 									{ isFreePlanAndCustomDomainFeature ? (
-										<Plans2023Tooltip
-											text={ translate( '%s is not included', {
-												args: [ domainName as string ],
-												comment: '%s is a domain name.',
-											} ) }
-										>
-											<FreePlanCustomDomainFeature
-												key={ key }
-												domainName={ domainName as string }
-											/>
-										</Plans2023Tooltip>
+										<FreePlanCustomDomainFeature key={ key } domainName={ domainName as string } />
```

Comment:
Good point. Initially I just did that thinking it might make it look cleaner. Relooking at it, leaving the tooltip there is actually better since it keeps the DOM structure of different conditional branches the same. I've brought it back :)